Source/WebCore/platform/graphics/skia/FontPlatformDataSkia.cpp

```diff
@@ -54,6 +54,45 @@ FontPlatformData::FontPlatformData(sk_sp<SkTypeface>&& typeface, float size, boo
     m_features = WTFMove(features);
 }
 
+FontPlatformData::FontPlatformData(float size, FontOrientation&& orientation, FontWidthVariant&& widthVariant, TextRenderingMode&& textRenderingMode, bool syntheticBold, bool syntheticOblique, RefPtr<FontCustomPlatformData>&& customPlatformData)
+    : FontPlatformData(size, syntheticBold, syntheticOblique, orientation, widthVariant, textRenderingMode, customPlatformData.get())
+{
+    m_font = SkFont(customPlatformData->m_typeface, m_size);
+    m_font.setEmbolden(m_syntheticBold);
+    m_font.setSkewX(m_syntheticOblique ? -SK_Scalar1 / 4 : 0);
+    m_font.setEdging(FontRenderOptions::singleton().antialias());
+    m_font.setHinting(FontRenderOptions::singleton().hinting());
+
+    m_hbFont = SkiaHarfBuzzFont::getOrCreate(*m_font.getTypeface());
+}
```

Comment:
What happens with the features? I think we could just use the existing constructor instead of duplicating the code here.